{"uuid": "891d95cb-68ba-46f2-8214-53546b8881b4", "children": ["381e6e1a-b935-43e6-9a1d-7ee73cf29118"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "AttributeError: 'Session' object has no attribute 'testfaied'\n", "trace": "  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 901, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\PycharmProjects\\selenium_project_14.09.2023\\page_object_pattern\\tests\\conftest.py\", line 15, in setup\n    before_failed = request.session.testfaied\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1694947202707, "stop": 1694947205074}], "start": 1694947202707, "stop": 1694947205109}